src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx

```diff
@@ -15,92 +15,111 @@ redirects:
 
-## View logs for your APM and infrastructure data [#logs-context]
-
-You can also bring your logs and application's data together to make troubleshooting easier and faster. With [logs in context](/docs/logs/logs-context/configure-logs-context-apm-agents/), you can see log messages related to your errors and traces directly in your app's UI. You can also see logs in context of your [infrastructure data](/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent/), such as Kubernetes clusters. No need to switch to another UI page.
-
-## How to integrate APM and infrastructure data [#how-it-works]
+The integration of [APM](/docs/apm/new-relic-apm/getting-started/welcome-new-relic-apm) and [infrastructure](/docs/infrastructure/new-relic-infrastructure/getting-started/welcome-new-relic-infrastructure) data allows you to see the health of your entire system from a single page. On the APM **Summary** page you can monitor hosts, apps, events, and alerts activity and use embedded change tracking to compare your data with any recent deployments. From one page you can respond to an alert, identify a root cause, and quickly resolve any impacts to host performance. 
 
-For <InlinePopover type="apm" /> and infrastructure data to be integrated, all of the following must be true:
+First, this doc will walk you through the process of resolving infrastructure issues with APM. Then it will dig deeper into some of the key features of APM and infrastructure monitoring. 
 
-* The APM agent and the infrastructure agent must be installed on the same host.
-* Both agents must use the same <InlinePopover type="licenseKey" />.
-* They must use the [same hostname](/docs/infrastructure/new-relic-infrastructure/troubleshooting/apm-data-missing-infrastructure#hostnames).
+<Steps>
+  <Step>
+    ## Integrate APM and infrastructure data [#how-it-works]
 
-If the integration is not working, see [Troubleshooting the APM-Infrastructure integration](/docs/infrastructure/new-relic-infrastructure/troubleshooting/apm-data-missing-infrastructure).
+    For <InlinePopover type="apm" /> and infrastructure data to be integrated, all of the following must be true:
 
-## View APM charts [#apm-chart]
+    * The APM agent and the infrastructure agent must be installed on the same host.
+    * Both agents must use the same <InlinePopover type="licenseKey" />.
+    * They must use the [same hostname](/docs/infrastructure/new-relic-infrastructure/troubleshooting/apm-data-missing-infrastructure#hostnames).
 
-When your <InlinePopover type="apm" /> and infrastructure data is linked, you have access to APM data charts on these Infrastructure monitoring UI pages: **Hosts**, **Network**, **Storage**, and **Processes**.
+If the integration is not working, see [Troubleshooting the APM-infrastructure integration](/docs/infrastructure/new-relic-infrastructure/troubleshooting/apm-data-missing-infrastructure).
```

Comment:
"If the integration" should probably be "if your integration", helps establish the link between the reader and the doc. I almost didn't notice this, the rest of the doc does a great job of doing that!